common/app/app/CommonServices.scala

```diff
@@ -0,0 +1,12 @@
+package app
+
+import com.softwaremill.macwire._
+import contentapi.{CapiHttpClient, ContentApiClient}
+import play.api.libs.ws.WSClient
+
+trait CommonServices {
```

Comment:
I'm not sure where I stand about that trait.
On one side `CommonServices` is generic enough to be dangerous as we may shove everything in here, on the other I'm not sure `ContentApiComponents` would be much better as it is very specific.
Do you think if you had a `ContentApiComponents` you would add more to it?